src/components/Expenses/ExpenseDate.tsx

```diff
@@ -1,6 +1,6 @@
 import "./ExpenseDate.css";
 
-function ExpenseDate(props) {
+function ExpenseDate(props: { date: Date }) {
```

Comment:
I'd actually destructure the props in here `props = {date}` instead. This way you don't type the unnecessary props all the time.